src/components/FilterBar.jsx

```diff
@@ -15,7 +15,8 @@ import sortOptions, { DEADLINE_ASC } from '../lib/sortOptions';
 import MajorFilter from './MajorFilter';
 import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
 
-const Filter = ({ title, filter }) => {
+const FilterButton = (props) => {
+  const { title } = props;
```

Comment:
You can still use the other form
```suggestion
const FilterButton = ({title, children}) => {
```